plugins/module_utils/pureport_client.py

```diff
@@ -8,6 +8,7 @@
     Client = None
     ClientHttpException = None
 
+API_URL="https://api.pureport.com"
```

Comment:
Likely want to remove this, the `API_URL` is imported above from the client.

Reply:
ugh good catch